### ISSUE TITLE
THEA withdrawal - balances.LiquidityRestrictions?

### ISSUE THREAD

**Issue by jaymansfield** [Polkadex-Substrate/Polkadex-Open-Beta#512]
Receiving a message "balances.LiquidityRestrictions" when trying to withdrawal even though I have sufficient balance.

I'm trying to withdrawal 0.5 GLMR

![image](https://github.com/Polkadex-Substrate/Polkadex-Open-Beta/assets/6218678/5d185152-8800-4a8c-867e-618ee59d2f48)

![image](https://github.com/Polkadex-Substrate/Polkadex-Open-Beta/assets/6218678/cd7e6d4f-1f47-4eb2-9cbf-f9060ca7eef9)
onchainaddress:{esm99RRxKvainDsRR6UHRcju4vNnAw1ouZEw64Yfn12tc6HpQ}

**Comment by pavankanteti** [comment]
Did you check if you have enough PDEX to pay the fee?

**Comment by jaymansfield** [comment]
Yes I have PDEX to cover gas and enough GLMR to cover the withdrawal. I'm not sure whats going wrong just tried again and its still not working. Still showing "balances.LiquidityRestrictions".

![image](https://github.com/Polkadex-Substrate/Polkadex-Open-Beta/assets/6218678/25327c2b-c28e-43bd-96d1-257a5ec2f083)

![image](https://github.com/Polkadex-Substrate/Polkadex-Open-Beta/assets/6218678/ff7a994b-f502-4ff6-b4af-25040d49047f)

![image](https://github.com/Polkadex-Substrate/Polkadex-Open-Beta/assets/6218678/b4e31e32-a5ea-4b37-a79c-225a8fb74d96)

**Comment by Gauthamastro** [comment]
> @pavankanteti Yes I have PDEX to cover gas and enough GLMR to cover the withdrawal. I'm not sure whats going wrong just tried again and its still not working. Still showing "balances.LiquidityRestrictions".
> 
> ![image](https://user-images.githubusercontent.com/6218678/244768772-25327c2b-c28e-43bd-96d1-257a5ec2f083.png)
> 
> ![image](https://user-images.githubusercontent.com/6218678/244768645-ff7a994b-f502-4ff6-b4af-25040d49047f.png)
> 
> ![image](https://user-images.githubusercontent.com/6218678/244768675-b4e31e32-a5ea-4b37-a79c-225a8fb74d96.png)

LiquidityRestrictions error is thrown when your PDEX is locked or staked and cannot be used for paying fees.

**Comment by jaymansfield** [comment]
@Gauthamastro @pavankanteti  Still not working.. I have 196 pdex and 0.74 GLMR (there are no locks according to THEA asset page, and its not staked this is the testnet).  I'm trying to withdrawal 0.5 GLMR.

Wallet address esnbKSRnU1WKNgJewte4asokmCWwiKBL2HeVWS3NSYEiZfoYc

**Comment by jaymansfield** [comment]
So I do have some locks according to polkadot.js. but i still have 5 pdex available so there should still be enough for the gas fee

![image](https://github.com/Polkadex-Substrate/Polkadex-Open-Beta/assets/6218678/8964a334-ac3f-45ca-8273-9878c759246f)
